src/components/common/AccountList.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <ul class="account-list">
+    <li v-for="account in accounts" :key="account.name" class="account">
+      <div class="account-info">
+        <h3>{{ account.name }}</h3>
+        <Bech32 :address="account.address || ''" />
+      </div>
+      <TmBtn
```

Comment:
This won't hide the Use Account button in the extension right?

Reply:
nope

Reply:
But shouldn't it?!